src/handlers/transfers/handler.js

```diff
@@ -39,30 +39,32 @@
 
 const Logger = require('@mojaloop/central-services-shared').Logger
 const TransferService = require('../../domain/transfer')
-const Utility = require('../lib/utility')
+const Util = require('../lib/utility')
 const Kafka = require('../lib/kafka')
 const Validator = require('./validator')
-const TransferState = require('../../lib/enum').TransferState
-const TransferStateEnum = require('../../lib/enum').TransferStateEnum
 const Enum = require('../../lib/enum')
-const TransferEventType = Enum.transferEventType
-const TransferEventAction = Enum.transferEventAction
+const TransferState = Enum.TransferState
+const TransferStateEnum = Enum.TransferStateEnum
+const TET = Enum.transferEventType
```

Comment:
For readability i would suggest changing this to transferEventTypeEnum or transferEventType

Reply:
ok